recipe/meta.yaml

```diff
@@ -2,17 +2,21 @@
 {% set version = "0.6.1" %}
 
 package:
-  name: {{ name|lower }}
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 517d5f9441d28c64b2bbdb87f4349e5751ba1fe920cd76faa7493688d5f40e6b
+  - folder: dist
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 517d5f9441d28c64b2bbdb87f4349e5751ba1fe920cd76faa7493688d5f40e6b
+  - folder: src
+    url: https://github.com/pydata/pydata-sphinx-theme/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: b7e0df298b71e5f2684744bc58539541d7ca09e03d36f12838a6a2259794d6a8
```

Comment:
We could also include the tests in the source distribution for the next release?